keyboards/mt84/rules.mk

```
@@ -0,0 +1,22 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration
```

The `yes` option is deprecated and is being replaced by `full`. Same functionality though.

```suggestion
BOOTMAGIC_ENABLE = full     # Virtual DIP switch configuration
```